apps/texttest/weekly/MeshConvert/Tutorial.wiki

```diff
@@ -15,19 +15,6 @@ our special binary version of the Triangle/Tetgen format.
  to containing element indices.  The presence of this file cuts the calculation cost at mesh load time.
  * If a Chaste format fibre file (.axi or .ortho) is associated with the mesh then this will also be converted to our binary format.
 
-=== Making the executable(s) ===
```

Comment:
This really wants replacing with cmake example rather than just deleting

Reply:
I thought naking standalone excutables wasn't possible anymore? @fcooper8472 ?

Reply:
We don't try and make them for people due to difficulties getting compatible cross-platform ones for Linux (used to generate the cardiac executable for download) but still should be able to make them (like ApPredict).

Reply:
Yes, it's still absolutely possible. We no longer ship a cardiac executable that you can expect to run portably without compiling it yourself, but the top level apps directory (and indeed the apps directory in any user project) can have standalone executables with their own `int main()` methods.

Reply:
ok but how do we write that in the tutorial?

Reply:
This would just be the corresponding instructions but using CMake, I suppose?
```
scons chaste_libs=1 compile_only=1 exe=1 build=GccOpt apps
```
would become, assuming you have configured Chaste
```
make Chaste
```
or
```
make MeshConvert
```
and the `build=GccOpt_ndebug` equivalent would be passing `-DCMAKE_BUILD_TYPE=Release` at configure time.